libs/books/feature/src/lib/book-search/books-grid/books-grid.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="books?.length">
+  <div class="book-grid">
+    <div class="book" data-testing="book-item" *ngFor="let b of books">
```

Comment:
Could you fold the styling of this div into the component itself?

Do you need this div at all?